packages/Telephony/Actions/SerialNumberInput.cs

```diff
@@ -48,10 +48,28 @@ public SerialNumberInput([CallerFilePath] string sourceFilePath = "", [CallerLin
         public BoolExpression AllowInterruptions { get; set; } = false;
 
         /// <summary>
-        /// Gets or sets the serial number pattern to use to decide when the dialog has aggregated the whole message.
+        /// Gets or sets if SerialNumberInput should accept alphabet characters. 
         /// </summary>
-        [JsonProperty("regexPattern")]
-        public StringExpression RegexPattern { get; set; }
```

Comment:
I'm happy you're removing this. Even though this is a source breaking change, I don't think anybody used it, and I cannot imagine that it worked correctly for any but trivial patterns.